packages/neovim/tsconfig.json

```diff
@@ -3,6 +3,6 @@
   "compilerOptions": {
     "outDir": "./lib"
   },
-  "include": ["src", "**/api/Buffer.test.ts"],
-  "exclude": ["node_modules", "__tests__", "**/*.test.ts", "examples", "lib"]
+  "include": ["src"],
+  "exclude": ["node_modules", "examples", "lib"]
```

Comment:
Nice, so we can close https://github.com/neovim/node-client/issues/352 ?

Reply:
Not really, still need to work on packages/integration-tests/.